<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTabWidget" name="tableauCRUD">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Ajouter</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>831</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Données employé :</string>
            </property>
            <widget class="QLabel" name="IDlabel">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>90</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>ID</string>
             </property>
            </widget>
            <widget class="QLabel" name="CINlabel">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>130</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>CIN</string>
             </property>
            </widget>
            <widget class="QLabel" name="nomlabel">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>170</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Nom</string>
             </property>
            </widget>
            <widget class="QLabel" name="prenomlabel">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>210</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Prénom</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="ID_le">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>100</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="CIN_le">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>140</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="nom_le">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>180</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="prenom_le">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>220</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLineEdit" name="poste_le">
             <property name="geometry">
              <rect>
               <x>520</x>
               <y>100</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="postelabel">
             <property name="geometry">
              <rect>
               <x>450</x>
               <y>90</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Poste</string>
             </property>
            </widget>
            <widget class="QLabel" name="CINlabel_2">
             <property name="geometry">
              <rect>
               <x>450</x>
               <y>170</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Salaire</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="adresse_le">
             <property name="geometry">
              <rect>
               <x>520</x>
               <y>140</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="adresselabel">
             <property name="geometry">
              <rect>
               <x>450</x>
               <y>130</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="text">
              <string>Adresse</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="salaire_le">
             <property name="geometry">
              <rect>
               <x>520</x>
               <y>180</y>
               <width>171</width>
               <height>22</height>
              </rect>
             </property>
            </widget>
            <widget class="QWidget" name="horizontalLayoutWidget_3">
             <property name="geometry">
              <rect>
               <x>460</x>
               <y>230</y>
               <width>271</width>
               <height>80</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_ajouterEmploye">
                <property name="text">
                 <string>Ajouter</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_modifierEmploye">
                <property name="text">
                 <string>Modifier</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Supprimer </string>
        </attribute>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>841</width>
           <height>401</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <widget class="QWidget" name="horizontalLayoutWidget">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>90</y>
               <width>501</width>
               <height>111</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>70</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="IDlabel_3">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="IDsupp_le"/>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>70</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="pushButton_supprimerEmploye">
             <property name="geometry">
              <rect>
               <x>350</x>
               <y>220</y>
               <width>120</width>
               <height>39</height>
              </rect>
             </property>
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Afficher</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>831</width>
           <height>401</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>22</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
